week2/1-tuesday/class/exercise11/exercise11.js

```diff
@@ -0,0 +1,18 @@
+const randomString = "Random String";
+
+function checkForSorE(string){
+    var lowerCase = string.toLowerCase();
```

Comment:
Very good, minor suggestion for error checking:

```suggestion
function checkForSorE(userInput) {
    if(typeof userInput !== "string") return `Sorry, you must enter a string, not a ${typeof userInput}`;
    var lowerCase = userInput.toLowerCase();
```

I've renamed `string` to `userInput` to make it easier to follow what we're doing when the input is not actually a `string`. It's fine to leave it how you did it but this might save someone a few seconds when they read it for the first time.